packages/muelu/src/Smoothers/MueLu_Ifpack2Smoother_def.hpp

```diff
@@ -136,7 +136,18 @@ namespace MueLu {
     }
     else if (type_ == "BLOCK RELAXATION" ||
              type_ == "BLOCK_RELAXATION" ||
-             type_ == "BLOCKRELAXATION")
+             type_ == "BLOCKRELAXATION" ||
```

Comment:
Y'all know that Ifpack2::Factory does this, right?

Reply:
It does, but not quite for the reason we're doing it here, which is to figure out if we need to give it coordinates.

Reply:
ah ok.